app.js

```diff
@@ -10,13 +10,14 @@ var ejs = require('ejs');
 var favicon = require('serve-favicon');
 var spawn = require('child_process').spawn;
 
-var url = 'mongodb://localhost:27017/query-server';
+//var url = 'mongodb://localhost:27017/query-server';
+var url = 'mongodb://user:pass@host:port/db'
```

Comment:
Hardcoding wont work. You'll have to read it from the ENV Variable set by Heroku